eZ/Publish/Core/MVC/Symfony/FieldType/Tests/User/ParameterProviderTest.php

```diff
@@ -44,6 +44,9 @@ protected function setUp(): void
         $this->parameterProvider = new ParameterProvider($this->userService);
     }
 
+    /**
+     * @requires PHP < 8.1
```

Comment:
The reason for that was most likely some regression coming from PHP 8.1, ref: https://github.com/ezsystems/ezplatform-kernel/runs/4354252098?check_suite_focus=true#step:7:156.